.github/workflows/swiftlint.yml

```diff
@@ -0,0 +1,21 @@
+name: SwiftLint
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/swiftlint.yml'
+      - '.swiftlint.yml'
+      - '**/*.swift'
+
+jobs:
+  SwiftLint:
+
```

Comment:
Nit: extra whitespace here and under `runs-on` that's inconsistent with the other actions.